apps/mobile/src/features/Signer/components/SignerView.tsx

```diff
@@ -89,16 +92,21 @@ export const SignerView = ({
             </YStack>
           </XStack>
         </Container>
+
+        {hasPrivateKey && !editMode && (
+          <View marginTop={'$4'} borderTopWidth={1} borderColor={'$borderLight'} paddingTop={'$4'}>
+            <SafeListItem
+              label="View private key"
+              rightNode={<SafeFontIcon name="chevron-right" />}
+              onPress={onPressViewPrivateKey}
+              pressStyle={{ opacity: 0.2 }}
+            />
+          </View>
+        )}
       </ScrollView>
       <KeyboardAvoidingView behavior="padding" keyboardVerticalOffset={top + bottom}>
         <View paddingHorizontal={'$4'} paddingTop={'$2'} paddingBottom={bottom ?? 60}>
-          {editMode ? (
-            <SafeButton onPress={onPressEdit}>Save</SafeButton>
-          ) : (
-            <SafeButton danger={true} onPress={onPressDelete}>
-              Remove signer
-            </SafeButton>
```

Comment:
I would suggest to offer a delete signer option either in the overview of signers or still here. Otherwise users have to open the signer, view the private key and can only then delete the signer. Lets do it in another PR though as discussed.